frontend/components/instructor/mobile/ButtonBar.tsx

```diff
@@ -1,3 +1,26 @@
-export default function ButtonBar() {
-  return <div>hi jeremy!!!!</div>;
+import Button from "../Button";
+
+interface ButtonBarProps {
+  primaryButtonText: string;
+  secondaryButtonText?: string;
+}
+
+export default function ButtonBar({
+  primaryButtonText,
+  secondaryButtonText,
+}: ButtonBarProps) {
+  return (
+    <div className="surface_border_tertiary fixed bottom-0 z-50 flex h-[65px] w-full items-center justify-center border-t-[1px] bg-white p-[8px]">
+      <div className="flex w-full  gap-[8px]">
+        {secondaryButtonText && (
+          <Button className="action_bg_secondary button-secondary h-full w-full rounded-[10px] border-t-[1px]">
```

Comment:
also what i was saying here was button-secondary should take care of everything except the hfull wfull i think?